src/utils/metrics.ts

```diff
@@ -69,9 +69,9 @@ export const reorgsTotal = new client.Counter({
   labelNames: ["chain_id"],
 });
 
-export const newContractsTotal = new client.Counter({
-  name: "watch_tower_new_contracts_total",
-  help: "Total number of new contracts",
+export const contractsTotal = new client.Counter({
```

Comment:
i don't love the previous nor the new suggestion. 
I feel the `contractsTotal` is also a bit generic. 

maybe `conditional_order_owners_total` is nicer?

Reply:
Actually i will change this in the PR (my first suggestion was `watch_tower_contracts_total`)